spharpy/samplings/coordinates.py

```diff
@@ -1,7 +1,8 @@
 import numpy as np
-from spharpy.samplings.helpers import sph2cart
+from pyfar.classes.coordinates import sph2cart, cyl2cart
+import pyfar as pf
+from spharpy.samplings.helpers import sph2cart as sp_sph2cart
```

Comment:
Why are you importing two different functions for the conversion here? Can't we stick to a single one?